client/my-sites/sites/sites.jsx

```diff
@@ -93,4 +108,13 @@ export default React.createClass( {
 			</Main>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state ) => ( {
+		selectedSite: getSelectedSite( state ),
+		isPrivateSite: ( site ) => isPrivateSite( state, site.ID ),
```

Comment:
You are right this construct doesn't sound right here although we use it everywhere!
I'll have a look but we might want to use `mergeProps` instead